README.md

```diff
@@ -1,43 +1,65 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
 
-## Usage
+Baseline image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
```

Comment:
This is not a baseline image unless I'm getting this wrong. 90% of your users won't extend this image. So you might as well just say, `This is the Docker image for ...`.